website/source/api/system/mounts.html.md

```diff
@@ -108,8 +103,7 @@ This endpoint enables a new secrets engine at the given path.
     These control the default and maximum lease time-to-live, force
     disabling backend caching, and option plugin name for plugin backends
```

Comment:
Remove reference to plugin name & backends.

Reply:
Good catch!